config/application.rb

```diff
@@ -27,15 +27,15 @@ class Application < Rails::Application
 
     # Set the number of seconds the timeout warning should occur before
     # login session is timed out.
-    config.session_timeout_warning_seconds = 150
+    config.session_timeout_warning_seconds = (Figaro.env.session_timeout_warning_seconds || 150).to_i
     # Set the number of seconds in which to delay the start of the
     # PeriodicalQuery() call. Make sure the sum of this value and
     # session_timeout_warning_seconds is a multiple of 60 seconds.
-    config.session_check_delay             = 30
+    config.session_check_delay             = (Figaro.env.session_check_delay || 30).to_i
     # Set the frequency of the PeriodicalQuery() call in seconds.
     # Make sure the sum of this value and session_timeout_warning_seconds
     # is a multiple of 60 seconds.
-    config.session_check_frequency         = 30
+    config.session_check_frequency         = (Figaro.env.session_check_frequence || 30).to_i
 
     config.middleware.use Rack::Attack
```

Comment:
What I meant by moving this to application.yml was taking all of this out of config/application.rb, such that `session_timeout_warning_helper.rb` points to application.yml instead of Rails.application.config